README.md

```diff
@@ -68,10 +68,26 @@ TileDB-ML can be installed:
 
       git clone https://github.com/TileDB-Inc/TileDB-ML.git
       cd TileDB-ML
-      pip install .
+  
+      # In case you want to install and check all frameworks. If you
+      # use zsh replace .[full] with .\[full\]
+      pip install -e .[full]
```

Comment:
We do not us this pattern in any other repo. Infact we've specifically designed against it. What is the motivation for this? Was there been a larger discussion on this or a feature doc or clubhouse story with logic you can link me to offline?

cc: @ihnorton